docs/reference/remap/functions/parse_query_string.cue

```diff
@@ -0,0 +1,38 @@
+package metadata
+
+remap: functions: parse_query_string: {
+	category: "Parse"
+	description: #"""
+		Parses the `value` as a query string.
+		"""#
+	notices: [
+		"""
+			All values are returned as strings. We recommend manually coercing values to desired types as you see fit. Empty keys and values are allowed.
+			""",
+	]
+
+	arguments: [
+		{
+			name:        "value"
+			description: "The string to parse."
+			required:    true
+			type: ["string"]
+		},
+	]
+	internal_failure_reasons: []
+	return: types: ["object"]
+
+	examples: [
+		{
+			title: "Parse query string"
+			source: #"""
+				parse_query_string("foo=%2B1&bar=2&bar=3&xyz")
```

Comment:
Also, should we strip `?` as a leading character for convenience? I can see that being a case that people might run into, and I don't see a particular downside to always doing so.

Reply:
Good idea! Done.